gimbal-adapter/src/main/java/com/urbanairship/gimbal/GimbalAdapter.java

```diff
@@ -207,27 +195,16 @@ public void removeListner(Listener listener) {
      */
     @RequiresPermission(ACCESS_FINE_LOCATION)
     public boolean start(@NonNull String gimbalApiKey) {
-        if (isStarted) {
-            return true;
-        }
-
-        Gimbal.setApiKey((Application) context.getApplicationContext(), gimbalApiKey);
-
-        if (!isPermissionGranted()) {
```

Comment:
If the permissions are applied after you call start it will start at that point.